src/status_im/contexts/profile/settings/view.cljs

```diff
@@ -9,18 +9,24 @@
             [status-im.contexts.profile.settings.list-items :as settings.items]
             [status-im.contexts.profile.settings.style :as style]
             [status-im.contexts.profile.utils :as profile.utils]
+            [status-im.feature-flags :as ff]
             [utils.debounce :as debounce]
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]))
 
+(defn show-settings-item?
+  [{:keys [feature-flag]}]
+  (or (ff/enabled? feature-flag)
```

Comment:
`ff/enabled?` should cater for `nil` values and return false.

Reply:
In this case we want to show the settings-item if they don't provide a feature flag, so the default case should be to show a settings-item and only hide ones that have been explicitly feature-flagged.